integration-tests/locales/all/pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>io.quarkus</groupId>
-        <artifactId>quarkus-integration-test-locales</artifactId>
+        <artifactId>quarkus-integration-test-locales-parent</artifactId>
```

Comment:
Is this part necessary?

Reply:
It's the same convention as used elsewhere